Unbox.swift

```diff
@@ -674,34 +666,78 @@ private extension Unboxer {
             throw UnboxError.InvalidData
         }
     }
+
+    static func performUnboxingWithDictionary<T: Unboxable>(dictionary: UnboxableDictionary, context: Any? = nil, allowInvalidElements: Bool = false) throws -> T {
+        return try Unboxer(dictionary: dictionary, context: context).performUnboxing(allowInvalidElements)
+    }
+
+    static func performUnboxingWithDictionaries<T: Unboxable>(dictionaries: [UnboxableDictionary], context: Any? = nil, allowInvalidElements: Bool = false) throws -> [T] {
+        return try dictionaries.map {
```

Comment:
Isn't this really the only point where we need the `allowInvalidElements` flag? Do we really need to pass it along into the actual unboxing process? Shouldn't we just do a `do try catch` inside the `map` closure, and if `allowInvalidElements` is true, we just silence the error?